README.md

```diff
@@ -275,6 +275,11 @@ git add .
 ```
 
 4. Run the test suite
+
+- open in Android Studio
+- navigate to kotlin/src/test/java/app.rive.runtime.kotlin/ExampleUnitTest
```

Comment:
I think the tests of interest are in `app.rive.runtime.kotlin.core`, where it has tests for a lot of the API stuff. The `ExampleUnitTest` looks to be just a sample unit test

Reply:
done